Trimble.Modus.Components/Handlers/EntryHandler.cs

```diff
@@ -27,6 +27,7 @@ public void MapTMEntry(IEntryHandler entryHandler, IEntry entry)
         
             entryHandler.PlatformView.BorderBrush = null;
             entryHandler.PlatformView.BorderThickness = new Microsoft.UI.Xaml.Thickness(0);
+            entryHandler.PlatformView.Padding = new Thickness(0).ToPlatform();

```

Comment:
This line is throwing the below mentioned error - 

Error	CS1929	
'Thickness' does not contain a definition for 'ToPlatform' and the best extension method overload 'PaintExtensions.ToPlatform(Paint)' requires a receiver of type 'Microsoft.Maui.Graphics.Paint'

Reply:
![ss](https://github.com/trimble-oss/modus-mobile-maui-components/assets/148334972/f9f9bb5b-7727-4cc6-8694-49103393a360)